[2025-01-31T00:00:12.350+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T00:00:12.392+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-01-30T23:57:22.460856+00:00 [queued]>
[2025-01-31T00:00:12.416+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-01-30T23:57:22.460856+00:00 [queued]>
[2025-01-31T00:00:12.417+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-01-31T00:00:12.443+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_permissions_dag> on 2025-01-30 23:57:22.460856+00:00
[2025-01-31T00:00:12.451+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=720) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T00:00:12.453+0000] {standard_task_runner.py:72} INFO - Started process 721 to run task
[2025-01-31T00:00:12.455+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_permissions_dag', 'manual__2025-01-30T23:57:22.460856+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpcdiwqq7v']
[2025-01-31T00:00:12.458+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask assign_permissions_dag
[2025-01-31T00:00:12.553+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-01-30T23:57:22.460856+00:00 [running]> on host 512f7139f01a
[2025-01-31T00:00:12.781+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_permissions_dag' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T23:57:22.460856+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-30T23:57:22.460856+00:00'
[2025-01-31T00:00:12.783+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T00:00:12.788+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-31T00:00:12.809+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T00:00:12.810+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=assign_permissions_dag, run_id=manual__2025-01-30T23:57:22.460856+00:00, execution_date=20250130T235722, start_date=20250131T000012, end_date=20250131T000012
[2025-01-31T00:00:12.871+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T00:00:12.923+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-31T00:00:12.926+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
